php-nginx/README.md

```diff
@@ -19,6 +19,15 @@ docker-compose build php56_nginx
 docker-compose push php56_nginx
 ```
 
+## About
+
+This is a Docker image for PHP (using the php-fpm SAPI) and Nginx. It uses, by
+default, our recommended default configuration, including:
+
+* A default HTTPS only website, with HTTP redirecting to HTTPS
```

Comment:
It was introduced in a previous PR, that PR supported HTTPS as well, just was disabled by default until at least self signed certs were available.

#61 https://github.com/continuouspipe/dockerfiles/pull/61/files#diff-079ac36fbe458f15bd03fc001098e215R44